bazel/conf/.bazelrc.build

```diff
@@ -143,3 +140,6 @@ common --noexperimental_inmemory_dotd_files
 # convention would also benefit. If the test does not support this, this "almost
 # certainly" does no harm.
 test --test_env=CLICOLOR_FORCE=true
+
+# Speed up compilation with zig cc
```

Comment:
out of curiosity, do you know how reliably the caching is? i.e. if an input changes, will it for sure perform a rebuild?

Reply:
There is [this disclaimer](https://github.com/uber/hermetic_cc_toolchain/tree/main?tab=readme-ov-file#zig-cache-location) in the hermetic cc readme:
> Currently zig cache is stored in /var/tmp/zig-cache, so bazel clean --expunge will not clear the zig cache. Zig's cache should be stored somewhere in the project's path. It is not clear how to do it.
> 
> See https://github.com/uber/hermetic_cc_toolchain/issues/83 for more context.

Here's the reference to this option speeding up the build, that I found: https://github.com/uber/hermetic_cc_toolchain/issues/134#issuecomment-1853868070

Reply:
Found this while getting upstream PRs ready: https://github.com/uber/hermetic_cc_toolchain/pull/183.